src/components/LicenseDropdown.vue

```diff
@@ -1,67 +1,69 @@
 <template>
```

Comment:
These changes should be reversed, as they are changing the code style to the style that's incompatible with repo linter settings. You can do this by running eslint: `npm run-script lint`.

Reply:
Hey @obulat, Thanks for the review. I have reversed the changes that were specific to Step.spec.js file, rest of the errors are related to "unit/utils/license-uilities.test.js"  file.